kirara_ai/ioc/container.py

```diff
@@ -46,11 +60,12 @@ def resolve(self, key: Any) -> Any: ...
 
     def resolve(self, key: Type[T] | Any) -> T | Any:
         """
-        依照{key}从容器获取一个值或对象实例。
+        依照{key}从容器解析出一个值或对象实例。
         如果{key}在当前容器中不存在，则会递归查找父容器。
 
         Args:
-            key: 对象的标识键
+            key: 对象的标识键, 一般为对象名如 DependencyContainer, LLMManager等, 
+                会根据类型自动查找对应对象实例。你也可以使用其他能被dict作为键的类型，但更推荐使用对象名。
```

Comment:
不要用字符串，因为很容易出现冲突。
使用 Type 基本上不会冲突。

```suggestion
            key: 对象的标识键, 一般为对象的类 (Type) 如 IMManager, LLMManager等, 
                会根据类型自动查找对应对象实例。
```